packages/components/src/components/Text/Text.stories.tsx

```diff
@@ -29,6 +29,7 @@ const meta: Meta<TextProps> = {
   argTypes: {
     /** The following condition hides the size props if variant's value is 'display' */
     size: { if: { arg: 'variant', neq: 'display' } },
+    bold: { if: { arg: 'variant', eq: 'body' } },
```

Comment:
```suggestion
    /** The following condition shows the `bold` prop if variant's value is `body` */
    bold: { if: { arg: 'variant', eq: 'body' } },
```

Or maybe a more generic explanation of how it works so one comment suffices without edit moving forward.

Reply:
Updated to use more generic explanation